File: httpsnippet/src/test/resources/output/javascript/fetch/multipart-data.js
let form = new FormData();
form.append("foo", "hello.txt");

const fetchOptions = {
  "mode": "cors",
  "method": "POST",
  "headers": {
    "content-type": "multipart/form-data"
  },
  "body": form
}

fetch("http://mockbin.com/har", fetchOptions)
 .then(response => response.json())
 .then(data => console.log(data))
 .catch(error => console.log(error));

File: httpsnippet/src/test/resources/output/c/libcurl/multipart-data.c
CURL *hnd = curl_easy_init();

curl_easy_setopt(hnd, CURLOPT_URL, "http://mockbin.com/har");

struct curl_slist *headers = NULL;
headers = curl_slist_append(headers, "content-type: multipart/form-data");
curl_easy_setopt(hnd, CURLOPT_HTTPHEADER, headers);

struct curl_httppost* post = NULL;
struct curl_httppost* last = NULL;

curl_formadd(&post, &last, CURLFORM_COPYNAME, "foo", CURLFORM_FILE, "hello.txt", CURLFORM_END)

curl_easy_setopt(hnd, CURLOPT_HTTPPOST, post)

CURLcode ret = curl_easy_perform(hnd);
